fern/pages/sdks/integration/maven-central.mdx

```diff
@@ -7,7 +7,7 @@ To authenticate and integrate Maven Central within your Java SDK, you'll need to
 ## Creating a Maven Central User Token
```

Comment:
Can we add a `<info>` below this and link to the original guide https://central.sonatype.org/publish/generate-token/ ?